content/en/docs/reference/command-line-tools-reference/feature-gates/persistent-volume-last-phase-transition-time.md

```diff
@@ -13,6 +13,10 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.29"
+    toVersion: "1.29"
```

Comment:
should this be "1.30"?

Reply:
Feature was beta for 2 releases so I think yes - corrected.